src/tests/Directory.Build.targets

```diff
@@ -271,11 +271,135 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ConvertCountOfStressMarkersToListPhase1" BeforeTargets="ConvertCountOfStressMarkersToListPhase1">
+    <PropertyGroup>
+      <SequenceOfSemicolons>$([System.String]::new(';',10))</SequenceOfSemicolons>
+    </PropertyGroup>
+  </Target>
+
   <Target Name="GenerateMarkerFiles" BeforeTargets="AssignTargetPaths">
+    <PropertyGroup Condition="'$(NumberOfStripesToUseInStress)' == '' and '$(IsMergedTestRunnerAssembly)' == 'true'">
+      <NumberOfStripesToUseInStress>1</NumberOfStripesToUseInStress> <!-- Set a custom number of stripes in stress to allow very large merged groups to function in the presence of GCStress -->
+    </PropertyGroup>
+
+    <!-- Generate a set of stress marker files to be used in stress scenarios in CI runs -->
+    <ItemGroup Condition="'$(NumberOfStripesToUseInStress)' != '' and '$(IsMergedTestRunnerAssembly)' == 'true'">
```

Comment:
We should add a backlog item to introduce an MSBuild task for this instead of hard-coding a list of 0-99.